source/hp/fe_values.cc

```diff
@@ -135,7 +135,8 @@ namespace hp
   template <int dim, int q_dim, class FEValuesType>
   FEValuesBase<dim, q_dim, FEValuesType>::FEValuesBase(
     const FEValuesBase<dim, q_dim, FEValuesType> &other)
-    : fe_collection(other.fe_collection)
+    : Subscriptor()
```

Comment:
```suggestion
    : Subscriptor(other)
```
We could also use the copy-constructor here to be more precise, but it [does essentially what the default constructor does](https://github.com/dealii/dealii/blob/bcd1a32da88cab396a3708cb3880049d1b98106b/include/deal.II/base/subscriptor.h#L277-L287). Which way do you prefer?

Reply:
Let's use the copy constructor here in case we change something at the `Subscriptor` class in the future.